src/server/modules/user/services/UrlManagementService.ts

```diff
@@ -142,6 +143,18 @@ export class UrlManagementService implements interfaces.UrlManagementService {
   ) => Promise<UrlsPaginated> = (conditions) => {
     return this.userRepository.findUrlsForUser(conditions)
   }
+
+  bulkCreate: (userId: number, urlMappings: BulkUrlMapping[]) => Promise<void> =
+    async (userId, urlMappings) => {
+      const result = await this.urlRepository.bulkCreate({
+        userId,
+        urlMappings,
+      })
+      dogstatsd.increment('shortlink.create', urlMappings.length, 1, [
+        `isbulk:true`,
+      ]) // TODO: extract metric and tag names
+      return result
```

Comment:
sorry I realise the return type is `Promise<void>` here, probably shouldn't need to return anything!
```suggestion
  bulkCreate: (userId: number, urlMappings: BulkUrlMapping[]) => Promise<void> =
    async (userId, urlMappings) => {
      await this.urlRepository.bulkCreate({
        userId,
        urlMappings,
      })
      dogstatsd.increment('shortlink.create', urlMappings.length, 1, [
        `isbulk:true`,
      ]) // TODO: extract metric and tag names
```